Formula/julia.rb

```diff
@@ -69,7 +73,7 @@ def install
       USE_SYSTEM_LAPACK=1
       USE_SYSTEM_GMP=1
       USE_SYSTEM_MPFR=1
-      USE_SYSTEM_LIBSUITESPARSE=1
+      USE_SYSTEM_LIBSUITESPARSE=0
```

Comment:
CC @carlocab @fxcoudert if any concern on using `julia`'s bundled copy of `suite-sparse`.

It doesn't look like HEAD supports SuiteSparse 6 yet, so I doubt we will get support in near future.

Reply:
Will remove `do not merge` in about 8 hours. Afterward, we can use any of the open `julia` PRs if an alternative approach is preferred.